frontend/src/translations/pt.json

```diff
@@ -63,8 +66,7 @@
         },
         "insufficientFunds": "Saldo insuficiente. Seu saldo é {{userInputTokenBalance}} {{assetSymbol}}",
         "moreThanMaximumWithdrawal": {
-          "sell": "O valor máximo de compra é {{maxAmountUnits}} {{assetSymbol}}.",
```

Comment:
I think the problem here is not a duplicate but the same key name, ie both being called "sell". Please re-add the key and call it "buy" instead.